test/test_fx.py

```diff
@@ -1054,8 +1054,8 @@ def forward(self, x):
         for node in tc_traced.graph.nodes:
             opcodes.add(node.op)
             if node.op == 'output':
-                output_shape = node.args[0].meta['shape']
-                output_stride = node.args[0].meta['stride']
+                output_shape = node.args[0].meta['tensor_meta'].shape
```

Comment:
Perhaps `tensor_meta` is redundant considering its on an attribute called `.meta`. Maybe just `meta['tensor']`?

Reply:
`meta['tensor']` can be interpreted as containing a tensor, I think `tensor_meta` is the right level of descriptiveness